src/test/system/lender/supply_redeem/redeem_two_tranche.t.sol

```diff
@@ -28,16 +38,171 @@ contract RedeemTwoTrancheTest is BaseSystemTest {
+
+        juniorInvestor.doSupply(jSupplyAmount);
+        seniorInvestor.doSupply(sSupplyAmount);
+
+        // new loan, should take all from junior and 60 from senior
+        uint ceiling = 100 ether;
+        uint rate = 1000000564701133626865910626; // 5% per day compound in seconds
+        uint speed = rate;
+        (uint loanId,) = createLoanAndWithdraw(borrower_, ceiling, rate, speed);
+
+        hevm.warp(now + 1 days);
+
+        uint seniorDebt = senior.debt();
+        assertEq(seniorDebt, 63 ether);
+
+        repayLoan(borrower_, loanId, seniorDebt + jSupplyAmount);
+        // junior redeem will break the minJuniorRatio
+        juniorInvestor.doRedeem(jSupplyAmount);
     }
 
-    function testSimpleRedeem() public {
-        uint investorBalance = 100 ether;
-        uint supplyAmount = 10 ether;
-        uint redeemAmount = supplyAmount;
-        supply(investorBalance, supplyAmount);
-        juniorInvestor.doRedeem(redeemAmount);
+    function testRedeemWithDefaults() public {
+        uint jSupplyAmount = 50 ether;
+        uint sSupplyAmount = 200 ether;
+
+        supplyFunds(sSupplyAmount, seniorInvestor_);
+        supplyFunds(jSupplyAmount, juniorInvestor_);
+
+        uint minJuniorRatio = 2 * 10**26;
+        FileLike(assessor).file("minJuniorRatio" , minJuniorRatio);
+
+        juniorInvestor.doSupply(jSupplyAmount);
+        seniorInvestor.doSupply(sSupplyAmount);
+
+        // new loan, should take all from junior and 50 from senior
+        uint ceiling = 100 ether;
+        uint rate = 1000000564701133626865910626; // 5% per day compound in seconds
+        uint speed = rate;
+        (uint loanA,) = createLoanAndWithdraw(borrower_, ceiling, rate, speed);
+
+        // new loan, should take 100 from senior
+        (uint loanB,) = createLoanAndWithdraw(borrower_, ceiling, rate, speed);
+
+        hevm.warp(now + 5 days);
+        // 5% senior rate with an interestBearingAmount of 150 ether
+        assertEq(senior.debt(), 191.442234375 ether);
+
+        // loan B has defaulted
+        uint threshold = 115 ether;
+        uint recoveryPrice = 75 ether;
+        addKeeperAndCollect(loanB, threshold, borrower_, recoveryPrice);
+        // 75 recovery price + 50 ether reserve
+        assertEq(currency.balanceOf(address(senior)), 125 ether);
+
+        // 100 * 1.05^5 = 127.62815625
+        uint loanDebt = 127.62815625 ether;
+        repayLoan(borrower_, loanA, loanDebt);
+
+        seniorInvestor.doRedeem(sSupplyAmount);
+        assertEq(senior.debt(), 0);
+        // 150 * (1.05^5) = 191.442234375, 41.442234375 ether profit
+
+        // balance should be initial investment = 200 ether  + profit
+        uint expectedSeniorBalance = 241.442234375 ether;
+        assertEq(currency.balanceOf(seniorInvestor_), expectedSeniorBalance);
+        juniorInvestor.doRedeem(jSupplyAmount);
+        // 63.814078125 ether was the expected profit for the second loan but junior takes the losses
```

Comment:
Can we rewrite the comment a bit?

Because from the tranches it is always a pool perspective and I would describe how you calculated the number `63.81`